projects.css

```diff
@@ -0,0 +1,205 @@
+.parent-container {
+  display: grid;
+  grid-template-rows: auto auto;
+  row-gap: 12px;
+  padding-right: 16px;
+  padding-left: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] The project card is styled correctly and matches almost perfectly with the template, but it misses a border and this way it is hard to tell the limits of the cards. Please add a border to make it match the Figma template and this section looks more professional.